app/test/prepare_proposal_test.go

```diff
@@ -28,22 +28,22 @@ func TestPrepareProposal(t *testing.T) {
 	testApp := testutil.SetupTestAppWithGenesisValSet(t)
 
 	type test struct {
-		input            abci.RequestPrepareProposal
-		expectedMessages []*core.Message
-		expectedTxs      int
+		input         abci.RequestPrepareProposal
+		expectedBlobs []core.Blob
+		expectedTxs   int
 	}
 
-	firstNS := []byte{2, 2, 2, 2, 2, 2, 2, 2}
+	firstNamespace := []byte{2, 2, 2, 2, 2, 2, 2, 2}
```

Comment:
[non-blocking][for future reference]
is this change related at all? its already completed, so **pls keep**, but if its 100% not related, in the future I would appreciate it if we try to not arbitrarily change variable names and put them into unrelated PRs. I don't mind this specific change, and I don't wan't to discourage renaming things, and I've been guilty of this too, but if any of us did it would be silly to block on an otherwise perfect PR.

thx :slightly_smiling_face:

Reply:
Agreed completely. This change isn't necessary in this PR. While working on this PR, I noticed that we use two names to refer to the same thing `ns` and `namespace`. I started correcting in this PR which was a mistake, should've been a new issue and PR.